src/Stormpath/Http/Authc/RequestSigner.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Stormpath\Http\Authc;
+
+/*
+ * Copyright 2013 Stormpath, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use Stormpath\ApiKey;
+use Stormpath\Http\Request;
+
+interface RequestSigner
+{
+    public function signRequest(Request $request, ApiKey $apiKey);
```

Comment:
Please add some documentation to this method.

Also, looks like this could be renamed to just `sign` as the name of the interface already implies that we are signing the request.